src/plugins/intel_gpu/tests/unit/test_cases/group_normalization_gpu_test.cpp

```diff
@@ -156,3 +163,61 @@ INSTANTIATE_TEST_SUITE_P(
         ::testing::ValuesIn({padding(), padding({0, 0, 1, 1})})));
 
 } // anonymous namespace
+
+#ifdef ENABLE_ONEDNN_FOR_GPU
+TEST(group_normalization, input_bfyx_output_fsv16) {
+    auto& engine = get_test_engine();
+
+    auto in_layout = layout{ ov::PartialShape{1, 3, 3, 2}, data_types::f32, format::bfyx };
+    auto scale_layout = layout{ ov::PartialShape{1, 1, 1, 1}, data_types::f32, format::bfyx };
+    auto bias_layout = layout{ ov::PartialShape{1, 1, 1, 1}, data_types::f32, format::bfyx };
+
+    auto input_mem = engine.allocate_memory(in_layout);
+    auto scale_mem = engine.allocate_memory(scale_layout);
+    auto bias_mem = engine.allocate_memory(bias_layout);
+
+    set_values(input_mem,
+               { 0.125, 0.125, 0.875, -0.125, 0.125, 0.750,
+                0.875, -0.375, -0.375, -1.000, -0.625, -1.000,
+                -0.125, -0.750, -0.250, 0.625, -0.500, -0.875 });
+    set_values(scale_mem, { 0.125 });
+    set_values(bias_mem, { 0.75 });
+
+    topology topology(
+        input_layout("input", in_layout),
+        input_layout("scale", scale_layout),
+        input_layout("bias", bias_layout),
+        group_normalization("group_normalization", input_info("input"), input_info("scale"), input_info("bias"), static_cast<std::int64_t>(1), 0.0025)
+    );
+
+    ExecutionConfig config = get_test_default_config(engine);
+    ov::intel_gpu::ImplementationDesc gn_impl = { format::bfyx, "", impl_types::ocl };
+    config.set_property(ov::intel_gpu::force_implementations(ov::intel_gpu::ImplForcingMap{{"group_normalization", gn_impl}}));
+    config.set_property(ov::intel_gpu::allow_new_shape_infer(true));
+    config.set_property(ov::intel_gpu::optimize_data(true));
+
+    auto network = std::make_shared<cldnn::network>(engine, topology, config);
+    network->set_input_data("input", input_mem);
+    network->set_input_data("scale", scale_mem);
+    network->set_input_data("bias", bias_mem);
+
+    auto outputs_gold = network->execute();
+    auto output_gold = outputs_gold.at("group_normalization").get_memory();
+    cldnn::mem_lock<float> output_mem_gold(output_gold, get_test_stream());
+
+    auto& gn_node = network->get_program()->get_node("group_normalization");
+    std::vector<layout> layouts = {layout { in_layout.get_shape(), data_types::f32, format::b_fs_yx_fsv16 }};
+    gn_node.set_output_layouts(layouts, false);
```

Comment:
This is not allowed behavior in our design flow. 
Once the  program is built, format should not be changed during runtime. 
Please create two separate networks.

Reply:
Separated the networks.